.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@david31982 @holgiosalos @JuanDelahozMiranda @manuelq12 @valeryibarra
```

Comment:
Add a new line to the end of file (EOF) in order to remove the warning from github:
![image](https://user-images.githubusercontent.com/10209180/166243428-064d9f29-f1cc-4427-a8b2-180a688c3831.png)